client/src/modules/accounts/accounts.html

```diff
@@ -1,37 +1,51 @@
-<div class="flex-header static">
+<div class="flex-header">
   <div class="bhima-title">
     <ol class="headercrumb">
       <li class="static">{{ "TREE.FINANCE" | translate }}</li>
       <li class="title">{{ "ACCOUNT.TITLE" | translate }}</li>
     </ol>
 
-   <div class="toolbar">
+    <div class="toolbar">
+      <fieldset ng-disabled="AccountsCtrl.transactions.isEditing()">
```

Comment:
I don't think this is right - `AccountsCtrl` shouldn't have a `transactions` property.